train.py

```diff
@@ -41,18 +41,18 @@ def val(args, model, dataloader, final_test):
     model_res_table = wandb.Table(columns=main_columns)
     
     print('start val!')
-    # label_info = get_label_info(csv_path)
     with torch.no_grad():
         model.eval()
         precision_record = []
         hist = np.zeros((args.num_classes, args.num_classes))
         for i, (data, label) in enumerate(dataloader):
             label = label.type(torch.LongTensor)
-            data = data.cuda()
-            label = label.long().cuda()
+            data = data
+            label = label.long()
```

Comment:
Ho visto che hai aggiunto la conversione in long già nella dataset class, quindi potrebbe essere inutile questa riga